gen/camt_056_001_08/camt_056_001_08.go

```diff
@@ -0,0 +1,405 @@
+// Code generated by xsdgen. DO NOT EDIT.
+
+package camt_056_001_08
+
+import (
+	"bytes"
+	"encoding/xml"
+	"time"
+
+	"github.com/moov-io/rtp20022/pkg/dt"
+)
+
+type ActiveOrHistoricCurrencyAndAmount struct {
+	Value float64                      `xml:",chardata"`
+	Ccy   ActiveOrHistoricCurrencyCode `xml:"Ccy,attr"`
+}
+
+// May be one of USD
+type ActiveOrHistoricCurrencyCode string
+
+type BranchAndFinancialInstitutionIdentification6 struct {
+	FinInstnId FinancialInstitutionIdentification18 `xml:"urn:iso:std:iso:20022:tech:xsd:camt.056.001.08 FinInstnId"`
+}
+
+type BranchAndFinancialInstitutionIdentification6TCH struct {
+	FinInstnId FinancialInstitutionIdentification18TCH `xml:"urn:iso:std:iso:20022:tech:xsd:camt.056.001.08 FinInstnId"`
+}
+
+type CancellationReason33Choice struct {
+	Cd    ExternalCancellationReason1Code `xml:"urn:iso:std:iso:20022:tech:xsd:camt.056.001.08 Cd,omitempty"`
+	Prtry Max35Text                       `xml:"urn:iso:std:iso:20022:tech:xsd:camt.056.001.08 Prtry,omitempty"`
```

Comment:
Did we follow the changes listed in the README?

Reply:
Oh whoops, I lost those changes somehow - pushed another commit